sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -338,18 +338,20 @@ private[spark] object HiveUtils extends Logging {
       }
 
       val classLoader = Utils.getContextOrSparkClassLoader
-      val jars = allJars(classLoader)
-      if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)) {
+      val jars: Array[URL] = if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)) {
         // Do nothing. The system classloader is no longer a URLClassLoader in Java 9,
         // so it won't match the case in allJars above. It no longer exposes URLs of
```

Comment:
Maybe remove the word 'above' in the comment.
Hm, so this resolves the problem? That's great. But I wonder why. In Java 9+, won't `allJars` already return nothing? I remember that was why I made this change to avoid the check for no JARs. But does it pick something up in some cases? OK makes some sense.

Reply:
+1 for removing `above` from `allJars above`.